src/templates/layouts/pressRelease/index.tsx

```diff
@@ -0,0 +1,155 @@
+                        window.print()
+                        return false
+                      }}
+                    >
+                      <i className="fa fa-print vads--u-padding-right--1"></i>
+                      Print
+                    </button>
+                    {pdfVersion && (
+                      <a href={pdfVersion.field_document.uri.url} download>
+                        <i className="fa fa-download vads--u-padding-right--1"></i>
+                        Download press release (PDF)
+                      </a>
+                    )}
+                  </div>
+                  <p className="va-introtext vads-u-font-size--lg vads-u-margin-top--3">
+                    {address.locality},{address.administrative_area} -
+                    {introText}
+                  </p>
+                  {/* Body */}
+                  <div dangerouslySetInnerHTML={{__html: fullText.processed}}></div>
+                </section>
+                <section className="vads-u-margin-bottom--6">
+                  <div className="vads-u-font-weight--bold">Media contacts</div>
+                  {/* Print each media contact */}
+                  {contacts.map((contact) => {
+                    if (!contact) {
+                      return null
+                    }
+                    return (
+                      <div key={contact.id}>
+                        <p className="vads-u-margin-top--1 vads-u-margin-bottom--0">
+                          {contact.title}{' '}
+                          {contact.field_description &&
+                            `, ${contact.field_description}`}
+                        </p>
+                        <p className="vads-u-margin-top--1 vads-u-margin-bottom--0">
+                          {contact.field_phone_number}
+                        </p>
+                        {contact.field_email_address && (
+                          <p className="vads-u-margin-top--1 vads-u-margin-bottom--0">
+                            <a href={`mailto:${contact.field_email_address}`}>
+                              {contact.field_email_address}
+                            </a>
+                          </p>
+                        )}
+                      </div>
+                    )
+                  })}
+                </section>
+                {downloads.length > 0 && (
+                  <section className="vads-u-margin-bottom--6">
+                    <div className="vads-u-font-weight--bold vades-u-margin-bottom--1">
+                      Download media assets
+                    </div>
+                    {/*Print out unorder list links per type*/}
+                    {downloads.map((asset) => {
+                      if (!asset) {
+                        return null
+                      }
+                      return (
+                        <ul key={asset.id}>
+                          <li>
+                            {asset.type === 'DrupalMediaDocument' && (
```

Comment:
if these downloads are normalized in the formatter, all of them could become `<a href={asset.url} download> {asset.name} </a>`